tests/unit/parks-dao.test.js

```diff
@@ -0,0 +1,162 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiExclude = require('chai-exclude');
+const _ = require('lodash');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire');
+
+const parksSerializer = appRoot.require('api/v1/serializers/parks-serializer');
+const { createConnStub } = require('./test-util');
+const {
+  fakeId,
+  fakeParkPostBody,
+  fakeParkPatchBody,
+  testCases,
+} = require('./test-data');
+
+chai.should();
+chai.use(chaiExclude);
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+let parksDao;
+const {
+  singleResultInList,
+  multiResult,
+  emptyResultInList,
+  singleResult,
+  emptyResult,
+  noRowsAffected,
+  rowAffected,
+  noOutId,
+  emptyBody,
+  undefinedBody,
+} = testCases;
+
+describe('Test parks-dao', () => {
+  beforeEach(() => {
+    const serializeParkStub = sinon.stub(parksSerializer, 'serializePark');
+    const serializeParksStub = sinon.stub(parksSerializer, 'serializeParks');
+    serializeParkStub.returnsArg(0);
+    serializeParksStub.returnsArg(0);
+
+    parksDao = proxyquire(`${appRoot}/api/v1/db/oracledb/parks-dao`, {
+      '../../serializers/parks-serializer': {
+        serializePark: serializeParkStub,
+        serializeParks: serializeParksStub,
+      },
+    });
+  });
+  afterEach(() => sinon.restore());
+
+  describe('Test getParks', () => {
```

Comment:
Is there any need to test queries for getParks? I don't believe its possible to get any invalid query params because of the openapi constraints.

Reply:
It depends on how much you want to test. You could pass in different query parameters and check that the sql query it uses is correct. Up to you.